src/frontend/mosh-server.cc

```diff
@@ -502,6 +517,14 @@ static int run_server( const char *desired_ip, const char *desired_port,
       exit( 1 );
     }
 
+    /* set SSH_AUTH_SOCK */
+    if ( agent.active() ) {
+      if ( setenv( "SSH_AUTH_SOCK", agent.listener_path().c_str(), true ) < 0 ) {
+	perror( "setenv" );
```

Comment:
given multiple setenv calls, it would be nice to upgrade to `setenv(SSH_AUTH_SOCK)` here, I think, and the corresponding change in, eg, the following perror.